scanomatic/image_analysis/first_pass_image.py

```diff
@@ -259,27 +260,24 @@ def run_marker_analysis(self, markings=None):
             ),
         )
 
-        im_analysis = image_fixture.FixtureImage(
+        im_analysis = image_fixture.FixtureImage.from_image(
             image=analysis_img,
             pattern_image_path=self["reference"].get_marker_path(),
             scale=scale_factor,
         )
 
-        (
-            x_positions_correct_scale,
-            y_positions_correct_scale
-         ) = im_analysis.find_pattern(markings=markings)
+        try:
+            (
+                x_positions_correct_scale,
+                y_positions_correct_scale
+            ) = im_analysis.find_pattern(markings=markings)
+
+        except FixtureImageError:
+            _logger.error("No markers found")
```

Comment:
should it really except this and continue? missing a return? or perhaps it is better that it crashes for now since without finding the pattern nothing really can be done.

Reply:
Hm you're right this wont work. I'll change so it crashes